src/components/ui/EditForm/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../Button';
+import EditIcon from '../EditIcon';
+import { convertFromPixelsToRem, inputValidation } from '../../../misc';
+
+interface EditFormProps {
+  handleSubmit: () => void;
+  handleInputBlur: () => void;
+  loading?: boolean;
+  imageLink?: string;
+  displayHeader?: boolean;
+  imageName?: string;
+  submitButtonName: string;
+  children: (props: any) => React.ReactNode;
+}
+
+function EditForm(props: EditFormProps): React.ReactElement<EditFormProps> {
+  const [values, setValues] = React.useState({});
+  const [errors, setErrors] = React.useState({});
+  const {
+    handleSubmit = (): void => null,
+    handleInputBlur = (): void => null,
+    loading = false,
+    imageLink = '',
+    displayHeader = true,
+    imageName = '',
+    submitButtonName = '',
+    children,
+  } = props;
+
+  function handleChange(event): void {
+    const {
+      target: { value },
+    } = event;
+    setValues(prevState => ({ ...prevState, value }));
```

Comment:
What is the name of the field whose value we are setting here?

Reply:
This can be used to set the value of any field in the form.

Reply:
it is reusable.

Reply:
The question is,  "What is the name of the input field we are setting there"
This is a short hand which amounts to `value: value`.